master/pkg/groupx/errgroupx/errgroupx.go

```diff
@@ -0,0 +1,45 @@
+package errgroupx
```

Comment:
it seems to me that errgroupx is strictly superior to waitgroupx: if you don't want error handling, just `.Go` with functions that return `nil` errors.

Reply:
yeah, i just didn't like throwing away the err with `_ := eg.Wait()` and a comment that "this can never happen"